csharp/extractor/Semmle.Extraction.CSharp/Analyser.cs

```diff
@@ -230,7 +230,7 @@ private void DoAnalyseCompilation(string cwd, string[] args)
                 var transformedAssemblyPath = PathTransformer.Transform(assemblyPath);
                 var assembly = compilation.Assembly;
                 var projectLayout = layout.LookupProjectOrDefault(transformedAssemblyPath);
-                var trapWriter = projectLayout.CreateTrapWriter(Logger, transformedAssemblyPath, true, options.TrapCompression);
+                var trapWriter = projectLayout.CreateTrapWriter(Logger, transformedAssemblyPath, options.TrapCompression, discardDuplicates: true);
```

Comment:
Wasn't this where we agreed it should be `discardDuplicates: false`?

Reply:
Ah, you're right. I messed up the rebase. Thanks for finding this.